api/v2.0/swagger.yaml

```diff
@@ -499,6 +499,68 @@ paths:
           $ref: '#/responses/409'
         '500':
           $ref: '#/responses/500'
+  /auditlogs:
```

Comment:
According to our internal REST API standard `URIs SHOULD use kebab-case`, `/audit-logs` is better

Reply:
updated, thx.